Lib/test/support/import_helper.py

```diff
@@ -274,3 +274,27 @@ def mock_register_at_fork(func):
     # memory.
     from unittest import mock
     return mock.patch('os.register_at_fork', create=True)(func)
+
+
+@contextlib.contextmanager
+def ready_to_import(name=None, source=""):
+    # To fix import cycles:
+    from test.support import script_helper
+
+    # sets up a temporary directory and removes it
+    # creates the module file
+    # temporarily clears the module from sys.modules (if any)
+    # reverts or removes the module when cleaning up
+    name = name or "spam"
+    with temp_dir() as tempdir:
```

Comment:
Is it really important to create a temporary directory?

Reply:
No idea :)

Reply:
So, I would prefer to keep it as-is (general semantics).